docs/cloud/azureai/flow-example-sharing-in-organization.md

```diff
@@ -0,0 +1,79 @@
+# Flow Example Sharing in Organization
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../../how-to-guides/faq.md#stable-vs-experimental).
+:::
+
+In this document, we will walk you through the steps to share your flow in an organization scope. This needs to register your flow as a model with some specific properties in an organization registry. Once completed, the model will be shown as a flow example in workspace portal.
+
+## Share flow as a model in an organization registry
+
+### Prepare a `readme.md` file for the flow
+
+To make the flow easily understand, it is better to include an `readme.md` file for the flow in the flow folder. Here we'll use [this existing readme](https://github.com/microsoft/promptflow/blob/main/examples/flows/chat/chat-with-wikipedia/README.md) as an example, the readme file may include following sections:
```

Comment:
Here is one way to rephrase that text for clarity and concision:

To make the flow easily understandable, include a README.md file in the flow folder explaining how to use it. The README may contain:
xxx

See this [example README](https://github.com/microsoft/promptflow/blob/main/examples/flows/chat/chat-with-wikipedia/README.md) with sections covering xxx. A good README improves discoverability and enables collaboration.